.github/workflows/ci.yml

```diff
@@ -5,6 +5,10 @@ on:
     types: [checks_requested]
   workflow_call:
     inputs:
+      image_prefix:
+        description: 'The prefix for the image name, "debug" or "perf"'
```

Comment:
Is "perf" the same as a release build, then?

Reply:
No, perf builds are debug Docker images + release binaries, so they include `iperf3` and `curl` for example.